### PR TITLE
[CAL-808] /availability/single - UI issue on buttons beside time inputs

### LINKED ISSUE #6518: [CAL-808] /availability/single - UI issue on buttons beside time inputs
**Current**
[https://user-images.githubusercontent.com/4536123/212856725-0ae1c87d-965c-459c-8b49-0b84cc8efdda.mp4](https://user-images.githubusercontent.com/4536123/212856725-0ae1c87d-965c-459c-8b49-0b84cc8efdda.mp4)

[d3e62f3d-6b35-4e7c-9698-f6bc92e7c782](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/93f21870-4e4e-42f7-a39c-05a4fb57031f/d3e62f3d-6b35-4e7c-9698-f6bc92e7c782)

**Issues**

* There's currently no spacing between icons buttons & inputs. Should be 8px
* The colour of icons in the buttons is inconsistent. Should be using [icon button component](https://ui.cal.com/?path=/story/button--all) from. [See here ](https://share.cleanshot.com/KVb7TllF)
* Remove "**Change the start and end times of your day"** title
* Fix copy in tooltips

**Should be**

[View in Figma](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=1671%3A112589&t=grPcOsn1FR52gxg2-1)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-808](https://linear.app/calcom/issue/CAL-808/availabilitysingle-ui-issue-on-buttons-beside-time-inputs)</sub>

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## What does this PR do?

- [x] Add space between icons buttons and inputs(8px)
- [x] Change color of icons(text-gray-700)
- [x] "Change the start and end times of your day" title removed 
- [x] Copy in tooltips changed to "Copy times to..."

Fixes #6518

**Environment**: Staging(main branch) / Production

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Demo
https://www.loom.com/share/5c1fee54539243ccb06c5024e5909980

### CODE REVIEW

File: packages/features/schedules/components/Schedule.tsx [hunk 1]
```diff
@@ -102,9 +102,12 @@ const CopyButton = ({
...
       <DropdownMenuTrigger asChild>
         <Button
-          className={classNames(open && "ring-brand-500 !bg-gray-100 outline-none ring-2 ring-offset-1")}
+          className={classNames(
+            "text-gray-700",
+            open && "ring-brand-500 !bg-gray-100 outline-none ring-2 ring-offset-1"
+          )}
           type="button"
-          tooltip={t("duplicate")}
+          tooltip={`${t("copy_times_to")}...`}
```
Comment (CarinaWolli) [inline packages/features/schedules/components/Schedule.tsx:110]: does that make sense in all languages, to have the dots in the end like that?

Comment (PeerRich) [reply packages/features/schedules/components/Schedule.tsx:110]: @CarinaWolli no, this has to be inside the string

Comment (PeerRich) [inline packages/features/schedules/components/Schedule.tsx:110]:
```suggestion
          tooltip={t("copy_times_to")}
```

Comment (PeerRich) [reply packages/features/schedules/components/Schedule.tsx:110]: also you need to add `copy_times_to` to en/common.json @gitstart-calcom

**Comment by Jaibles** [comment]
Loom looks great ❤️

File: apps/web/public/static/locales/en/common.json [hunk 2]
```diff
@@ -660,6 +660,7 @@
   "edit_availability": "Edit availability",
   "configure_availability": "Configure times when you are available for bookings.",
   "copy_times_to": "Copy times to",
+  "copy_times_to...": "Copy times to...",
```
Comment (PeerRich) [inline apps/web/public/static/locales/en/common.json:663]:
```suggestion
  "copy_times_to": "Copy times to …",
```

File: packages/features/schedules/components/Schedule.tsx [hunk 3]
```diff
@@ -102,9 +102,12 @@ const CopyButton = ({
...
       <DropdownMenuTrigger asChild>
         <Button
-          className={classNames(open && "ring-brand-500 !bg-gray-100 outline-none ring-2 ring-offset-1")}
+          className={classNames(
+            "text-gray-700",
+            open && "ring-brand-500 !bg-gray-100 outline-none ring-2 ring-offset-1"
+          )}
           type="button"
-          tooltip={t("duplicate")}
+          tooltip={t("copy_times_to...")}
```
Comment (PeerRich) [inline packages/features/schedules/components/Schedule.tsx:110]:
```suggestion
          tooltip={t("copy_times_to")}
```

File: apps/web/public/static/locales/en/common.json (hunk 2 above)
Comment (PeerRich) [reply apps/web/public/static/locales/en/common.json:663]: sorry for the confusion

**Review by PeerRich — APPROVED** [review]

**Review by CarinaWolli — APPROVED** [review]

**Pull request merged**